dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/router/tag/TagRouter.java

```diff
@@ -212,6 +214,9 @@ private boolean checkAddressMatch(List<String> addresses, String host, int port)
                 if (NetUtils.matchIpExpression(address, host, port)) {
                     return true;
                 }
+                if ((ALL_IP_MATCH + ":" + port).equals(address)) {
```

Comment:
Reuse CommonConstants.ANYHOST would be better.

Reply:
ok